autograder/autograder/insert_database_version_data.py

```diff
@@ -301,6 +300,5 @@ def get_result_details(data_dir, semester, course, g_id, who_id, version):
             result_json = json.load(result_file)
             # a = datetime.strptime(result_json[-1]['submission_time'], "%a %b  %d %H:%M:%S %Z %Y")
             a = dateutils.read_submitty_date(result_json[-1]['submission_time'])
-            result_details['submission_time'] = '{}-{:02d}-{:02d} {:02d}:{:02d}:{:02d}' \
-                .format(a.year, a.month, a.day, a.hour, a.minute, a.second)
+            result_details['submission_time'] = a.strftime('%Y-%m-%d %H:%M:%S%z')
```

Comment:
Submission timestamp is impacting not only this single python file, #1197 is the original PR that introduced `submission_time` so please review the PR and make it consistent and make sure it is not harming any of the functionality.